AUTHORS

```diff
@@ -983,7 +989,99 @@ Yuta Hiroto <git@about-hiroppy.com>
+phette23 <phette23@gmail.com>
+William Luo <toyota790@gmail.com>
+justshiv <siobhan@justshiv.com>
+Kevin Donahue <kevindonahuedevelopment@gmail.com>
+Jenna Vuong <hello@jennavuong.com>
+Anchika Agarwal <anchika.agarwal@gmail.com>
+Jean Regisser <jean.regisser@gmail.com>
+Fangshi He <hefangshi@gmail.com>
+Blake Embrey <hello@blakeembrey.com>
+Alex J Burke <alex@alexjeffburke.com>
+Faiz Halde <faiz_84123@yahoo.com>
+Rumkin <dev@rumk.in>
+Arve Seljebu <arve.seljebu@gmail.com>
+Devon Rifkin <drifkin@drifkin.net>
+Doug Wade <doug@dougwade.io>
+Mohsen <mohsenekhtiari@yahoo.com>
+Marian <sorca_marian@yahoo.com>
+Justin Sprigg <IPWNDYOU@gmail.com>
+Bryan Hughes <brhugh@microsoft.com>
+Ehsan Akhgari <ehsan@mozilla.com>
+Ingvar Stepanyan <me@rreverser.com>
+Adrian Estrada <edsadr@gmail.com>
+Matt Lavin <matt.lavin@gmail.com>
+Joao Andrade <jjadonotenter@gmail.com>
+Bartosz Sosnowski <bartosz@janeasystems.com>
+Nicolas Romer <nicolasromer@gmail.com>
+David A. Wheeler <dwheeler@dwheeler.com>
+Italo A. Casas <me@italoacasas.com>
+Jonathan Montane <jon@stratus.one>
+Ilkka Myller <ilkka.myller@nodefield.com>
+kimown <kimown@163.com>
+Natanael Copa <ncopa@alpinelinux.org>
+David Halls <dahalls@gmail.com>
+Greyson Parrelli <greyson.parrelli@gmail.com>
+Daniel Bevenius <daniel.bevenius@gmail.com>
+Bryce Simonds <bryce.simonds@q2ebanking.com>
+Tushar Mathur <tusharmath@gmail.com>
+Kyle E. Mitchell <kyle@kemitchell.com>
+akki <akki@users.noreply.github.com>
+Josh Gavant <josh.gavant@outlook.com>
+sartrey <sartrey@163.com>
+Guy Fraser <guy.fraser1@gmail.com>
+Ruslan Iusupov <rus0000@users.noreply.github.com>
+Michael Wain <michael.wain@skybettingandgaming.com>
+Zach Bjornson <bjornson@stanford.edu>
+Andras <andras@kinvey.com>
+Chuck Langford <field.email@gmail.com>
+Ryan Lewis <ryanharrisonlewis@gmail.com>
+Tarun Garg <tarungarg546@gmail.com>
+Diosney Sarmiento <diosney.s@gmail.com>
+Quentin Headen <qheaden@phaseshiftsoftware.com>
+Alex Hultman <alexhultman@localhost.localdomain>
+saadq <saad@saadquadri.com>
+Hargobind S. Khalsa <khalsah@gmail.com>
+Joran Siu <joransiu@ca.ibm.com>
+Vitaly Tomilov <vitaly-t@users.noreply.github.com>
+ratikesh9 <ratikesh92@gmail.com>
+Alex Perkins <aperkin@rei.com>
+Bethany N Griggs <Bethany.Griggs@uk.ibm.com>
+Joe Esposito <joe@joeyespo.com>
+Erin Spiceland <yes@erin.codes>
+Ravindra barthwal <ravindrabarthwal@users.noreply.github.com>
+Joey Cozza <joeycozza@gmail.com>
+Vladimir de Turckheim <vlad2t@hotmail.com>
+hugnosis <hugnosis@gmail.com>
+kibertoad <iselwin@gmail.com>
+Pat Pannuto <pat.pannuto@gmail.com>
+Haojian Wu <hokein.wu@gmail.com>
+Alhadis <gardnerjohng@gmail.com>
+Aleksei Koziatinskii <ak239spb@gmail.com>
+Adrian Nitu <adrian.nitu@intel.com>
+Ben Gourley <bn@grly.me>
+Kunal Pathak <Kunal.Pathak@microsoft.com>
+Tracy Hinds <tracyhinds@gmail.com>
+lazlojuly <lazlo.gy@gmail.com>
+BlackYoup <a.lefebvre@outlook.fr>
+not-an-aardvark <not-an-aardvark@users.noreply.github.com>
```

Comment:
That email is fine if that's what you'd like. How about your name? Do you want it left as `not-an-aardvark` or do you want your regular name listed (assuming it's not `not-an-aardvark` ;-) ...)